docs/README.md

```diff
@@ -49,7 +49,7 @@
 - [Brew Test Bot](Brew-Test-Bot.md)
 - [Diagram Guidelines](Diagram-Guidelines.md)
 - [Prose Style Guidelines](Prose-Style-Guidelines.md)
-- [Type Checking with Sorbet](Typechecking.md)
+- [Type Checking with Sorbet](Type-Checking.md)
```

Comment:
I think it would be good to avoid a rename to avoid breaking existing links. If renaming is still desired: can a redirect be setup and have it done in a new PR? It's hard to review in this one what the actual changes are to the file. Thanks!